rules-tests/CodeQuality/Rector/Ternary/SimplifyTautologyTernaryRector/Fixture/fixture.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
this file seems fixture file .php that commited, shoukd be removed

Reply:
Ok :+1:  Done in follow up PR